packages/encode/bench/compare.encode.bench.ts

```
@@ -0,0 +1,37 @@
+import { Base64NodeJs } from '@httpx/encode/base64/base64.nodejs';
```

ESLint

[error] Resolve error: typescript with invalid interface loaded as resolver

(import-x/namespace)